NEWS.md

```diff
@@ -5,6 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.9.x] – 2024-01-18
+
+### Added
+
+* added the real symplectic Grassmann manifold `SymplecticGrassmann`
+* Introduce the manifold of `HamiltonianMatrices` and a wrapper for `Hamiltonian` matrices
+* introduce `rand(:HamiltonianMatrices)`
+* extend `rand` to also `rand!` for `HamiltonianMatrices`, `SymplecticMatrices` and `SymplecticStiefel`
+* implement `riemannian_gradient` conversion for `SymplecticMatrices` and `SymplecticGrassmann`
+* Rename (deprecate) `Symplectic` to `SimplecticMatrices` in order to have a `Symplectic` wrapper for such matrices as well
+* Rename (deprecate) `SymplecticMatrix` to `SymplecticElement` to clarify that it is the special matrix ``J_{2n}`` and not an arbitrary symplectic matrix.
```

Comment:
These should be in a "Deprecated" section.

Reply:
Well-spotted, will check with the official guideline we follow.

Reply:
You are right, it also reads better that way. Fixed it.